ui/src/composables/private/__tests__/use-validate.cy.js

```diff
@@ -1,32 +1,123 @@
 import FieldWrapper from './FieldWrapper.vue'
 import Icons from '../../../../icon-set/material-icons.mjs'
+import { ref } from 'vue'
+import { vModelAdapter } from '@quasar/quasar-app-extension-testing-e2e-cypress'
 
 // const snapshotOptions = { customSnapshotsDir: '../__tests__' }
 
+function mountQFieldWrapper (options) {
+  return cy.mount(FieldWrapper, options)
+}
+
+function getHostElement () {
+  return cy.get('.select-root')
+}
+
 describe('use-validate API', () => {
   describe('Props', () => {
     describe('Category: behavior', () => {
       describe('(prop): error', () => {
-        it.skip(' ', () => {
-          //
+        it('should mark the field as having an error', () => {
+          mountQFieldWrapper({
+            error: true
+          })
+          getHostElement().should('not.have.class', 'q-field--error')
+
+          getHostElement().then(() => {
+            Cypress.vueWrapper.setProps({ error: true })
+          })
+
+          getHostElement().should('have.class', 'q-field--error')
         })
       })
 
       describe('(prop): rules', () => {
-        it.skip(' ', () => {
-          //
+        it('should validate value using custom validation logic', () => {
+          const errorMessage = 'Select value should be greater than 10 characters'
+          const model = ref('Option 1')
+          const options = [ 'Option 1', 'Option 2' ]
+
+          mountQFieldWrapper({
+            props: {
+              ...vModelAdapter(model),
+              options,
+              rules: [ val => val.length > 10 || errorMessage ]
+            }
+          })
+
+          getHostElement().then(() => {
+            Cypress.vueWrapper.setProps({ modelValue: 'Option 2' })
```

Comment:
When leveraging `vModelAdapter`, you can just use the `ref` as you would normally do with Vue

`model.value = 'Option 2'` should work exactly the same and is more idiomatic, as that's what's happening anyways under the hood:
https://github.com/quasarframework/quasar-testing/blob/0f92f9e095eb56942ad63fb4a98c5d15e0af0319/packages/e2e-cypress/src/helpers/v-model-adapter.ts#L16

You probably misinterpreted one of my previous comments
`vModelAdapter` is meant to make it easier and more idiomatic both changing a v-model prop and making assertions on it

See the following link about the latter case, even if it doesn't apply to this test in particular in which you're testing for DOM-related changes, and thus the bit is where the value ended up, not the value by itself

https://github.com/quasarframework/quasar-testing/blob/0f92f9e095eb56942ad63fb4a98c5d15e0af0319/packages/e2e-cypress/src/templates/typescript/src/components/___tests__/VModelComponent.cy.ts#L53-L72